[DOCS] Removes filesystem and core block config docs (#7913)
great-expectations/great_expectations: docs/docusaurus/docs/terms/data_connector.md

The Wildcard Data Reference is how Data Connectors deal with that problem, making it easy to search external stores and understand data.

> When defining a Data Connector for your Datasource, you may include wildcard Data References as part of the configuration for the Datasource.  This is done by including wildcards in the default regex defined in the Data Connector's portion of the Datasource's configuration.  Typically, you will see this used for `InferredAssetFilesystemDataConnector`s in Datasources connecting to a filesystem.  For an example of this, please see [our guide on how to connect to data on a filesystem using Pandas](/docs/0.15.50/guides/connecting_to_your_data/filesystem/pandas).

Reviewer: ```suggestion
When defining a Data Connector for your Datasource, you can include wildcard Data References as part of the Datasource configuration. To include wildcard Data References, you add the wildcards to the default regex string in the Data Connector's portion of the Datasource's configuration.  Typically, wildcard Data References are used for `InferredAssetFilesystemDataConnector`s in Datasources connecting to a filesystem.  For an example implementation, see [How to connect to data on a filesystem using Pandas](/docs/0.15.50/guides/connecting_to_your_data/filesystem/pandas).
```
